man/os-release.xml

```diff
@@ -392,6 +392,35 @@
           for light blue, or <literal>ANSI_COLOR="0;38;2;60;110;180"</literal> for Fedora blue.
           </para></listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>VENDOR_NAME=</varname></term>
+
+          <listitem><para>The name of the OS vendor. This is the name of the organization or company which
+          produces the OS. This field is optional.</para>
+
+          <para>This name is intended to be exposed in "About this system" UIs or software update UIs when
+          needed to distinguish the OS vendor from the OS itself.</para>
+
+          <para>Examples: <literal>VENDOR_NAME="Red Hat"</literal> for Red Hat Enterprise Linux,
```

Comment:
is "for" the right word here? wouldn't "on" be better, or so?

i mean, i am just wondering if people might assume that "Red Hat" was an abbreviation of sorts for "Red Hat Enterprise Linux", if you say "for", no?

or maybe remove the rhel example, just keep the canonical/ubuntu example, because people could not possibly misunderstand that as an abbreviation?

Reply:
FWIW, "for" seems fine to me. "Red Hat is the vendor *for* Red Hat Enterprise Linux". OTOH, "on" doesn't fit so well, because I'd say "When on the Red Hat Enterprise Linux system, Red Hat is the vendor", but that's more complicated for little benefit.

Reply:
(i am not a native English speaker, but @pwithnall is, so let's just do whatever @pwithnall thinks is right)

Reply:
‘for’ is the right word to use there. I’ve changed the example from Red Hat/RHEL to Fedora Project/Fedora Linux.